examples/schelling/model.py

```diff
@@ -64,9 +75,10 @@ def __init__(self, width=20, height=20, density=0.8, minority_pc=0.2, homophily=
             if self.random.random() < self.density:
                 agent_type = 1 if self.random.random() < self.minority_pc else 0
 
-                agent = SchellingAgent((x, y), self, agent_type)
+                agent = SchellingAgent(pos=(x, y), model=self, agent_type=agent_type, unique_id=agent_id)
```

Comment:
[The Schelling model in the benchmark](https://github.com/projectmesa/mesa/blob/6dbe3e8572db4fa0b0db6fee1ee89d4123587ad4/benchmarks/Schelling/schelling.py#L64) has a more concise way of generating the `agent_id`. Also, in general, the benchmark code for Flocking, Schelling, and WolfSheep, written by @quaquel, are more modern than the examples in this repo, and have yet to be synced/incorporated into this repo.

Reply:
Should I work on syncing them to this repo? There is the model and agent class in [the benchmark code](https://github.com/projectmesa/mesa/blob/main/benchmarks/Schelling/schelling.py) but it still needs to be integrated with the visualization

Reply:
Yes it would be great if the benchmark versions are integrated back into the example folder so they are again consistent.